chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/ProductController.java

```diff
@@ -316,4 +427,52 @@
         }
         return new ResponseEntity<SplitProductResponse>(response, responseHeaders, HttpStatus.OK);
     }
+
+    private class DuplicateChplProdNumber {
```

Comment:
Should this be a separate class since it's used in both developer and product controllers? It's hard to tell in this view but I think they are identical?